test/parallel/test-find-package-json.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const tmpdir = require('../common/tmpdir');
+const assert = require('node:assert');
+const fs = require('node:fs');
+const { findPackageJSON } = require('node:module');
+const { describe, it } = require('node:test');
+
+
+describe('findPackageJSON', () => { // Throws when no arguments are provided
+  it('should throw when no arguments are provided', () => {
+    assert.throws(
+      () => findPackageJSON(),
+      { code: 'ERR_MISSING_ARGS' }
+    );
+  });
+
+  it('should throw when parentLocation is invalid', () => {
+    for (const invalid of [null, {}, [], Symbol(), () => {}, true, false, 1, 0]) {
+      assert.throws(
+        () => findPackageJSON('', invalid),
+        { code: 'ERR_INVALID_ARG_TYPE' },
+      );
+    }
+  });
+
+  it('should accept a file URL (string), like from `import.meta.resolve()`', () => {
+    const importMetaUrl = `${fixtures.fileURL('whatever.ext')}`;
+    const specifierBase = './packages/root-types-field/';
+    assert.strictEqual(
+      findPackageJSON(`${specifierBase}index.js`, importMetaUrl),
+      fixtures.path(specifierBase, 'package.json')
```

Comment:
You probably need to wrap that
```suggestion
      path.toNamespacedPath(fixtures.path(specifierBase, 'package.json')),
```

Reply:
OH! sorry, I missed this. thank you 🙏